src/test/java/de/rabea/player/UnbeatableComputerPlayerTest.java

```diff
@@ -0,0 +1,100 @@
+package de.rabea.player;
+
+import de.rabea.game.Board;
+import de.rabea.game.Mark;
+import org.junit.Before;
+import org.junit.Test;
+
+import static de.rabea.game.Mark.*;
+import static org.junit.Assert.*;
+
+public class UnbeatableComputerPlayerTest {
+    UnbeatableComputerPlayer unbeatableComputerPlayer;
+    Board board;
+
+    @Before
+    public void setup() {
+        board = new Board(new Mark[] {EMPTY, EMPTY, EMPTY,
+                                      EMPTY, EMPTY, EMPTY,
+                                      EMPTY, EMPTY, EMPTY});
+        unbeatableComputerPlayer = new UnbeatableComputerPlayer(O);
```

Comment:
You can initialise `unbeatableComputerPlayer` directly in the declaration in line 12.